arch/arm/boot/dts/armada-385-ts7800-v2.dts

```diff
@@ -72,6 +72,25 @@
 		};
 	};
 
+	isa16550:  isa16550 { /* drivers/tty/serial/8250/ts7800_isa16550.c */
+                compatible = "technologicsystems,isa16550";
+                #address-cells = <0x1>;
+                #size-cells = <0x0>;
+                reg = <0xFA000000 0x400>;
+                isa16550,nports = <4>;   /* assuming a TS-SER4 */
+                status = "okay";
+        };
+
+        ax88796b: ax88796b {
+                compatible = "asix,ax88796b";
+                #address-cells = <0x1>;
+                #size-cells = <0x0>;
+                reg = <0xFA000000 400>; /* For the TS-ETH2 board on PC-104 */
```

Comment:
Is this hardcoded address going to cause issues? e.g. is this ultimately over the PCIe bus and may change location?